client/src/app/employee/employee-list/state/employee-list.model.ts

```diff
@@ -1,32 +1,22 @@
-export interface Employee {
+export interface EmployeeList {
```

Comment:
This reads, to me, like a custom type alias for `Employee[]`, but instead its the shape of employee data for the `employees` endpoint.

Maybe there's a better name?

In other apps I've worked on we've used `ListEmployee` for this, or `Employee` and `EmployeeDetail` to represent the minimal and full schemas.

Reply:
I like this naming convention: `EmployeeListItem`, `Employee`, and `EmployeeDetail`. Updated the list code to use this pattern.